config/vufind/RateLimiter.yaml

```diff
@@ -71,6 +71,18 @@ Storage:
 #                         query        Query string parameters
 #                         post         POST request parameters
 #
+# groupByIpv4Octets     Share a single quota of tokens between a group of related IP
+#                       addresses, defined by truncating the client IP. Useful for
+#                       botnets that control a range of IPs.  This config defines
+#                       the group by truncating the IP address, if it is IPv4, to the
+#                       given number of octets.  For example, setting this to 3 would
+#                       truncate 1.2.3.4 to 1.2.3 for the purpose of grouping.
```

Comment:
Perhaps add a mention that you should only use values 1-3 to avoid extra logic in address handling (and similar for IPv6)?

Reply:
Yup, larger numbers would work fine but not 0 or negative.  I added valid values to the docs.  (Technically 4 is also valid for ipv4 but would be pointless, so I agree with saying 1-3.)